ckanext/canada/views.py

```diff
@@ -0,0 +1,960 @@
+import json
```

Comment:
Would you leave this file name the same so that the changes made are more clear in this PR?

Reply:
reverted refactor of `view.py` -> `views.py`